Makefile

```diff
@@ -11,6 +11,7 @@ BUILD_AGENT=local
 TELEMETRY_REPORT_PERIOD = 24h # also configured in goreleaser.yml
 GW_API_VERSION = 1.0.0
 INSTALL_WEBHOOK = false
+TRACE_ON = false # generate a stack trace for unit-test failures. Helpful in CI environments.
```

Comment:
Is this too noisy to leave on as default?

Reply:
Actually, no, it's not that bad. I was just following the ginkgo docs suggestion only to use it in the CI pipeline. I'll just set it to true in the makefile command.